modules/Hoot/tools/clipDataset.js

```diff
@@ -208,16 +210,23 @@ export default class ClipDataset {
             let row         = d3.select( `#row-${ mapId }` ),
                 datasetName = row.select( '.datasetName' ),
                 outputName  = row.select( '.outputName' ),
-                pathName    = row.select( '.outputPath' );
+                pathName    = row.select( '.outputPath' ).property( 'value' ),
```

Comment:
I feel like the folders when chosen from the UI should set their id, rather than have to lookup by name.